visualization/app/codeCharta/ui/codeMap/codeMap.label.service.ts

```diff
@@ -31,7 +33,7 @@ export class CodeMapLabelService implements CameraChangeSubscriber {
 		ThreeOrbitControlsService.subscribe(this.$rootScope, this)
 	}
 
-	addLabel(node: Node) {
+	addLabel(node: Node, showNodeName: boolean, showMetricNameValue: boolean) {
```

Comment:
Suggestion:
To avoid something like that addLabel(true, false, null, ...)
We could introduce an options object: addLabel(node: Node, options: { showNodeName: boolean, showMetricNameValue: boolean})

The call would then be much nicer, I think:
addLabel(node, { showNodeName: false, showMetricNameValue: true })